app/qml/main.qml

```diff
@@ -274,10 +282,12 @@ ApplicationWindow {
       }
 
       MMToolbarButton {
+        id: addButton
+
         text: qsTr("Add")
         iconSource: __style.addIcon
         onClicked: {
-          if ( __recordingLayersModel.rowCount() > 0 ) {
+          if ( recordingLayersModel.rowCount() > 0 ) {
```

Comment:
Would be better to not instantiate `LayersProxyModel` just for this check, but instead add a function like `__activeProject.projectHasRecordingLayers()`